frontend/src/components/Secrets/GSecretDialogDDNS.vue

```diff
@@ -0,0 +1,245 @@
+          @update:model-value="v$.zone.$touch()"
+          @blur="v$.zone.$touch()"
+        />
+      </div>
+      <div>
+        <v-select
+          v-model="tsigSecretAlgorithm"
+          color="primary"
+          label="TSIG Secret Algorithm"
+          :items="secretAlgorithmItems"
+          variant="underlined"
+        />
+      </div>
+    </template>
+
+    <template #help-slot>
+      <div>
+        <p>
+          This DNS provider allows you to create and manage DNS entries for authoritive DNS server supporting dynamic updates with DNS messages following
+          <g-external-link url="https://datatracker.ietf.org/doc/html/rfc2136">
+            RFC2136
+          </g-external-link>
+          (DNS Update) like knot-dns or others.
+        </p>
+        <p>
+          The configuration is depending on the DNS server product. You need permissions for <code>update</code> and <code>transfer</code> (AXFR) actions on your zones and a TSIG secret.
+        </p>
+        <p>
+          For details see <g-external-link url="https://github.com/gardener/external-dns-management/tree/master/docs/rfc2136">
+            Gardener RFC2136 DNS Provider Documentation
+          </g-external-link>
+        </p>
+      </div>
+    </template>
+  </g-secret-dialog>
+</template>
+
+<script>
+import { useVuelidate } from '@vuelidate/core'
+import { required } from '@vuelidate/validators'
+
+import GSecretDialog from '@/components/Secrets/GSecretDialog'
+import GExternalLink from '@/components/GExternalLink'
+
+import { getErrorMessages } from '@/utils'
+import {
+  withFieldName,
+  withMessage,
+} from '@/utils/validators'
+
+export default {
+  components: {
+    GSecretDialog,
+    GExternalLink,
+  },
+  props: {
+    modelValue: {
+      type: Boolean,
+      required: true,
+    },
+    secret: {
+      type: Object,
+    },
+  },
+  emits: [
+    'update:modelValue',
+  ],
+  setup () {
+    return {
+      v$: useVuelidate(),
+    }
+  },
+  data () {
+    return {
+      server: undefined,
+      tsigKeyName: undefined,
+      tsigSecret: undefined,
+      zone: undefined,
+      tsigSecretAlgorithm: 'hmac-sha256',
+      hideTSIGSecret: true,
+      secretAlgorithmItems: [
+        {
+          title: 'HMAC-SHA256 (default)',
+          value: 'hmac-sha256',
+        },
+        {
+          title: 'HMAC-SHA224',
+          value: 'hmac-sha224',
+        },
+        {
+          title: 'HMAC-SHA384',
+          value: 'hmac-sha384',
+        },
+        {
+          title: 'HMAC-SHA512',
+          value: 'hmac-sha512',
+        },
+        {
+          title: 'HMAC-SHA1',
+          value: 'hmac-sha1',
+        },
+        {
+          title: 'HMAC-MD5',
+          value: 'hmac-md5',
+        },
+      ],
+    }
+  },
+  validations () {
+    return {
+      server: withFieldName('Server', {
+        required,
+        format: withMessage('Must have format <host>:<port>', value => {
+          return /^([^:]+):(\d+)$/.test(value)
```

Comment:
Is port mandatory?

Reply:
@MartinWeindel

Reply:
Good point. The port is optional. I need also to fix the documentation in the external-dns-management project.

Reply:
I removed the check